backend.py

```diff
@@ -22,6 +23,11 @@ class CardNotKnownError(LookupError):
     """Raised when a card doesn't belong to any known type."""
 
 
+# Load of robots displayed names in file robots.yaml
+with open('robots.yaml', encoding='utf-8') as file:
```

Comment:
```suggestion
with open('robots.yaml', encoding='utf-8') as robots_file:
```
(`file` je patrne jedno z klicovych slov Pythonu, soude podle barvy. Udelala bych to tedy podobne jako jsme uz delaly s `type`)

Reply:
> `file` je patrne jedno z klicovych slov Pythonu

Pouze v Pythonu 2. Máš staré obarvování :)
(Já ho mám taky staré – autoři barvítek většinou nedělají rozdíl mezi py2 a py3.)
Na používání proměnné `file` už není nic špatného. Ale na `robots_file` taky ne.